spec/spec.md

```diff
@@ -140,8 +140,8 @@ a text value, and terminated by a newline.
 The `SPDX-Copyright` tag MUST be followed by a copyright notice as described
 in <!-- FIXME: Refer to an internal section -->.
 
-Instead of the `SPDX-Copyright` tag, the word `Copyright` or the symbol `©` MAY
-be used, in which case a colon is not needed.
+Instead of the `SPDX-Copyright` tag, the symbol `©`, or the word `Copyright` MAY
+be used, in which case a colon is not needed..
```

Comment:
Unneeded extra dot at the end of line 144, but otherwise looks great and thanks, @carmenbianca  :)

Reply:
Oops, this is what I get for copy-pasting without verifying.

Will fix, thanks